docs/source/en/intro/egg-and-koa.md

```diff
@@ -3,12 +3,12 @@ title: Egg and Koa
 
 ## Asynchronous Programming Model
 
-Node.js is an asynchronous world, asynchronous programming models in official API support are all in callback form ，it brings many problems. For example:
+Node.js is an asynchronous world, asynchronous programming models in official API support are all in callback form，it brings many problems. For example:
 
 - [callback hell](http://callbackhell.com/): Notorious "callback hell"。
 - [release zalgo](https://oren.github.io/#/articles/zalgo/): Asynchronous functions may call callback function response data synchronously which would bring inconsistency.
 
-The community has provided many solutions for the problems, the winner is Promise, it is built into ECMAScript 2015. On the basis of Promise, and with the ability of Generator to switch context, we can write asynchronous code in synchronous way with [co] and other third party libraries. Meanwhile [async function], the official solution has been published in ECMAScript 2017 and landed in Node.js 8.
+The community has provided many solutions for the problems and the winner is Promise. It is built into ECMAScript 2015. On the basis of Promise, and with the ability of Generator to switch context, we can write asynchronous code in synchronous way with [co] and other third-party libraries. Meanwhile [async function], the official solution has been published in ECMAScript 2017 and landed in Node.js v8.
```

Comment:
not v8, it means 8.x

Reply:
Sure will keep the previous changes. Should this go into another commit or in the same commit with amend?

Reply:
in the same pr.  

we use `squash and merge`, so don't need to amend, just push commits

Reply:
Updated the PR.